src/interface.ts

```diff
@@ -72,4 +101,11 @@ export interface ITransactionNext {
     guardian: string;
     signature: Uint8Array;
     guardianSignature: Uint8Array;
-  }
+}
+
+export interface IMessage {
```

Comment:
Maybe here we can live without the interface (related to the brainstorming on sdk-py).

Thus, maybe the DTO should suffice, for now? Then introduce an interface when needed? CC: @ccorcoveanu, as well.

Reply:
Removed the interface.